modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue

```diff
@@ -543,16 +582,16 @@ export default {
           this.editMode = false;
         }
         this.draftMode = mode;
-        this.moduleOptions.context = doc;
+        this.moduleOptions.context = modeDoc;
         // Changes the ending from :published to :draft, etc.
-        this.moduleOptions.contextId = doc._id;
-        this.refreshOrReload(doc._url);
+        this.moduleOptions.contextId = modeDoc._id;
+        this.refreshOrReload(modeDoc._url);
       } catch (e) {
         if (e.status === 404) {
           // TODO don't get this far, check this in advance and disable it in the UI
           await apos.alert({
-            heading: 'Not Yet Published',
-            description: 'This document has never been published.'
+            heading: 'Does Not Exist Yet',
+            description: `That document is not yet available as ${mode} in the ${locale} locale.`
```

Comment:
Do editors know what a "locale" is? I don't know what the better term would be, but it seems technical.